lib/runners/ocaml.js

```diff
@@ -4,10 +4,46 @@ module.exports.run = function run(opts, cb) {
     shovel.start(opts, cb, {
         solutionOnly: function (run) {
             run({
-                name: 'ocaml',
-                args: ['-stdin'],
+                name: 'utop',
+                args: ['-require', 'core', '-require', 'batteries', '-require', 'oUnit', '-stdin'],
                 stdin: opts.solution
             });
+        },
+        testIntegration: function(run) {
+            var runFixtureUsingOUnit = [
+                'open OUnit',
+                'let cw_print_test_event = function',
+                '    | EStart (name::rest) -> print_endline ("<IT::>" ^ OUnit.string_of_node name)',
+                '    | EResult result -> ',
+                '        begin match result with ',
+                '        | RSuccess _ -> print_endline ("<PASSED::>Test passed")',
+                '        | RFailure (_, err) -> print_endline ("<FAILED::>" ^ err)',
```

Comment:
You probably will need to wrap print_endline with a function that replaces `\n` with `<:LF:>`, since otherwise multiline text will get interpreted as separate data (first line would be considered a failure, all other lines would be treated as logs).